### PR TITLE
op-guide, media: add pd metrics

### PR DESCRIPTION
**Pull request by rleungx**

### CODE REVIEW

File: op-guide/dashboard-pd-info.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    - Leader Balance Ratio： Leader 数量最多和最少节点相差的百分比，一般小于 5%，节点重启时会有比较大的波动
+    - Region Balance Ratio： Region 数量最多和最少节点相差的百分比，一般小于 5%，新增/下线节点时相差比较大
+    - Normal Stores: 处于正常状态的节点数目
+    - Abnormal Stores： 处于异常状态的节点数目，正常情况应当为 0
+    - Current Storage Usage： TiDB 集群存储空间的使用率
+    - Current Peer Count： 当前集群 peer 的总量
+    - Metadata Information： 记录集群 ID,时间戳和生成的 ID
+    - Region Label Isolation Level： 不同 label 所在的 level 的 Region 数量
+    - Region Health： 每个 Region 的状态，通常情况下，pending 的 peer 应该少于 100， miss 的 peer 不能一直大于 0
+- Balance |
```
Comment (disksing) [inline op-guide/dashboard-pd-info.md]:
```suggestion
- Balance
```

File: op-guide/dashboard-pd-info.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    - Raft Applied Index： 最后一次 apply 的 Raft index
+- TiDB 
+    - handle_requests_count： TiDB 的请求数量
+    - handle_requests_duration_seconds： 每个请求所花费的时间，99%的情况下，应该小于 100ms
+- Heartbeat
+    - Region heartbeat report： TiKV 向 PD 发送的心跳个数
+    - Region heartbeat report error： TiKV 向 PD 发送的异常的心跳个数
+    - Region heartbeat report active： TiKV 向 PD 发送的正常的心跳个数
+    - Region schedule push： PD 向 TiKV 发送的调度命令的个数
+    - 99% region heartbeat latency： 99%的情况下，心跳的延迟
```
Comment (disksing) [inline op-guide/dashboard-pd-info.md]:
```suggestion
    - 99% region heartbeat latency： 99% 的情况下，心跳的延迟
```

File: op-guide/dashboard-pd-info.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
+---
+title: 重要监控指标详解
```
Comment (lilin90) [inline op-guide/dashboard-pd-info.md]:
```suggestion
title: PD 重要监控指标详解
```

File: op-guide/dashboard-pd-info.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+category: monitoring
+---
+
+# 重要监控指标详解
```
Comment (lilin90) [inline op-guide/dashboard-pd-info.md]:
```suggestion
# PD 重要监控指标详解
```

File: op-guide/dashboard-pd-info.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+使用 Ansible 部署 TiDB 集群时，一键部署监控系统 (Prometheus/Grafana)，监控架构请看 [TiDB 监控框架概述](monitor-overview.md)。
```
Comment (lilin90) [inline op-guide/dashboard-pd-info.md]:
```suggestion
使用 Ansible 部署 TiDB 集群时，一键部署监控系统 (Prometheus/Grafana)，监控架构请看 [TiDB 监控框架概述](../op-guide/monitor-overview.md)。
```

File: op-guide/dashboard-pd-info.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+目前 Grafana Dashboard 整体分为 PD、TiDB、TiKV、Node\_exporter、Overview 等。
+
+对于日常运维，我们通过观察 PD 面板上的 Metrics，可以了解 PD 当前的状态。
+
+以下为 PD Dashboard 监控说明：
+
+## 说明
+
+- Cluster
+    - PD Role： 当前 PD 的角色
```
Comment (lilin90) [inline op-guide/dashboard-pd-info.md]: Please remove the extra space after the Chinese colon. And also remove similar ones involved in this PR.

File: op-guide/dashboard-pd-info.md [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+    - Storage Capacity： TiDB 集群总可用数据库空间大小
+    - Current Storage Size： TiDB 集群目前已用数据库空间大小
+    - Number of Regions： 当前集群的 Region 总量
+    - Leader Balance Ratio： Leader 数量最多和最少节点相差的百分比，一般小于 5%，节点重启时会有比较大的波动
+    - Region Balance Ratio： Region 数量最多和最少节点相差的百分比，一般小于 5%，新增/下线节点时相差比较大
+    - Normal Stores: 处于正常状态的节点数目
```
Comment (lilin90) [inline op-guide/dashboard-pd-info.md]: Change the colon to the Chinese version and remove the space after the colon.

File: op-guide/dashboard-pd-info.md [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+    - Schedule operator finish： 已完成的 operator 的数量
+    - Schedule operator timeout： 已超时的 operator 的数量
+    - Schedule operator replaced or canceled： 已取消或者被替换的 operator 的数量
+    - Schedule operators count by state： 不同状态的 operator 的数量
+    - 99% operator finish duration： 已完成的 operator 中，99% 所需花费的时间
+    - 50% operator finish duration： 已完成的 operator 中，50% 所需花费的时间
+    - 99% operator step duration： 已完成的 operator 的步骤中，99% 所需花费的时间
+    - 50% operator step duration： 已完成的 operator 的步骤中，50% 所需花费的时间
+- Grpc
+    - completed commands rate： grpc 命令的完成速率
```
Comment (lilin90) [inline op-guide/dashboard-pd-info.md]:
```suggestion
    - completed commands rate：gRPC 命令的完成速率
```

**Comment by lilin90** [comment]
I'll add this doc to README.md later.

File: op-guide/dashboard-pd-info.md [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+    - 99% handle_txns_duration_seconds： 99% 的情况下，处理 etcd 事务所需花费的时间
+    - 99% wal_fsync_duration_seconds： 99% 的情况下，持久化 WAL 所需花费的时间，这个值通常应该小于 1s
+    - 99% peer_round_trip_time_seconds： 99% 的情况下，etcd 的网络延时，这个值通常应该小于 1s
+    - etcd disk wal fsync rate： etcd 持久化 WAL 的速率
+    - Raft Term： 当前 Raft 的 term
+    - Raft Committed Index： 最后一次 commit 的 Raft index
+    - Raft Applied Index： 最后一次 apply 的 Raft index
+- TiDB 
+    - handle_requests_count： TiDB 的请求数量
+    - handle_requests_duration_seconds： 每个请求所花费的时间，99%的情况下，应该小于 100ms
```
Comment (nolouch) [inline op-guide/dashboard-pd-info.md]:
```suggestion
    - handle_requests_duration_seconds： 每个请求所花费的时间，99% 的情况下，应该小于 100ms
```

File: op-guide/dashboard-pd-info.md [hunk 10]
```diff
@@ -0,0 +1,99 @@
...
+    - Number of Regions：当前集群的 Region 总量
+    - Leader Balance Ratio：Leader 数量最多和最少节点相差的百分比，一般小于 5%，节点重启时会有比较大的波动
+    - Region Balance Ratio：Region 数量最多和最少节点相差的百分比，一般小于 5%，新增/下线节点时相差比较大
+    - Normal Stores：处于正常状态的节点数目
+    - Abnormal Stores：处于异常状态的节点数目，正常情况应当为 0
+    - Current Storage Usage：TiDB 集群存储空间的使用率
+    - Current Peer Count：当前集群 peer 的总量
+    - Metadata Information：记录集群 ID,时间戳和生成的 ID
+    - Region Label Isolation Level：不同 label 所在的 level 的 Region 数量
+    - Region Health：每个 Region 的状态，通常情况下，pending 的 peer 应该少于 100， miss 的 peer 不能一直大于 0
```
Comment (disksing) [inline op-guide/dashboard-pd-info.md]:
```suggestion
    - Region Health：每个 Region 的状态，通常情况下，pending 的 peer 应该少于 100，miss 的 peer 不能一直大于 0
```

**Comment by rleungx** [comment]
PTAL @nolouch @lilin90

**Pull request merged**